### PR TITLE
WIP - in app chat POC

### PR DESCRIPTION
**Pull request by owocki**
https://gitcoincore.slack.com/archives/CMGLVLSJX/p1567569729098000

dont merge; this is just a playground at this point..

### CODE REVIEW

File: app/retail/templates/shared/footer_scripts_lite.html [hunk 1]
```diff
@@ -41,6 +41,17 @@
 <script src="{% static "v2/js/lib/jquery.cookie.js" %}"></script>
 <script src="{% static "v2/js/lib/jquery.select2.min.js" %}"></script>
 <script src="{% static "v2/js/lib/moment.min.js" %}"></script>
+  <div id="sb_widget"></div>
+<script src="{% static "v2/js/chat.js" %}"></script>
+<script src="{% static "v2/js/chatwidget.js" %}"></script>
+<script>
```
Comment (octavioamu) [inline app/retail/templates/shared/footer_scripts_lite.html]: maybe add if document.contxt['github_handle'] then > code so just picking the logged users?

Comment (owocki) [reply app/retail/templates/shared/footer_scripts_lite.html]: agree; that should go in the prod build

**Comment by octavioamu** [comment]
Why??? 😢

**Comment by danlipert** [comment]
> Why??? 😢

I thought this looked cool as well

**Comment by owocki** [comment]
its like $900 a month at our activity level. we can move forward with it; not sure if this provider is the right one tho

could also BUIDL it and open source it (theres not a lot of open source options out there) i've built chat apps in the past. its a fun little websocket adventure. high request volume that scales with DAUs

**Comment by danlipert** [comment]
ahh bummer! thats ridiculously expensive!

**Pull request closed without merging**